tests/test_scheduler_local.py

```
@@ -0,0 +1,369 @@
+        assert "T" in job.next_fire  # ISO
+
+    def test_iso_one_shot(self, tmp_path):
+        s = _make_scheduler(tmp_path)
+        future = "2099-01-01T00:00:00"
+        job = s.add_job("hi", future)
+        # Naive ISO should be normalized to UTC
+        assert job.next_fire.startswith("2099-01-01T00:00:00")
+
+    def test_empty_prompt_rejected(self, tmp_path):
+        s = _make_scheduler(tmp_path)
+        with pytest.raises(ValueError):
```

Ruff (0.15.12)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)